modules/perception/onboard/transform_wrapper/transform_wrapper.cc

```diff
@@ -47,14 +47,13 @@ void TransformCache::AddTransform(const StampedTransform& transform) {
     return;
   }
 
-  while (!transforms_.empty()) {
+  do {
     delt = transform.timestamp - transforms_.front().timestamp;
     if (delt > cache_duration_) {
       transforms_.pop_front();
-    } else {
-      break;
     }
-  }
+    break;
+  } while (!transforms_.empty());
 
```

Comment:
It may fail if the transforms_ is initially empty. Also the logic seems incorrect.

Reply:
Thanks for your review! You are right, it should not break as will terminate the loop in this case. 👍 
It seems transforms_ will not be empty since we do this check at the front of this method.
I'll close this PR for now.